lutris/util/display.py

```diff
@@ -130,37 +131,120 @@ def get_display_manager():
 USE_DRI_PRIME = len(_get_graphics_adapters()) > 1
 
 
-def get_compositor_commands():
-    """Nominated for the worst function in lutris"""
+class DE(enum.Enum):
```

Comment:
DE is not a valid class name
Also, you don't seem to be using a code editor that tells you that DE is not a valid class name.

Reply:
Rename to DesktopEnvironment.

Reply:
In my defense, yes I am using a code editor that doesn't show `DE` as an invalid class name (Visual Studio Code), but only because the underlying linter (pylint) does not report the class name as invalid.
I also ran flake8 on the file and got no errors, and I ran `make test` which threw no errors.

I will change the class name now, give me a few seconds.

Reply:
That is weird, Pylint should totally report this as an invalid name.